google-cloud-bigtable/src/main/resources/changestream.json

```diff
@@ -0,0 +1,1232 @@
+            "commit_timestamp": "2022-07-01T00:00:00Z",
+            "tiebreaker": 100,
+            "chunks": [
+              {
+                "mutation": {
+                  "delete_from_family": {
+                    "family_name": "family"
+                  }
+                }
+              }
+            ]
+          }
+        },
+        {
+          "data_change": {
+            "type": "CONTINUATION",
+            "token": "data-change-token",
+            "low_watermark": "2022-07-01T00:00:00Z",
+            "chunks": [
+              {
+                "mutation": {
+                  "delete_from_family": {
+                    "family_name": "family"
+                  }
+                }
+              }
+            ],
+            "done": true
+          }
+        },
+        {
+          "data_change": {
+            "row_key": "0000000000000000",
+            "type": "USER",
+            "source_cluster_id": "source-cluster-id",
+            "commit_timestamp": "2022-07-01T00:00:00Z",
+            "tiebreaker": 100,
+            "chunks": [
+              {
+                "mutation": {
+                  "delete_from_family": {
+                    "family_name": "family"
+                  }
+                }
+              }
+            ]
+          }
+        },
+        {
+          "data_change": {
+            "type": "CONTINUATION",
+            "token": "data-change-token",
+            "low_watermark": "2022-07-01T00:00:00Z",
+            "chunks": [
+              {
+                "mutation": {
+                  "delete_from_family": {
+                    "family_name": "family"
+                  }
+                }
+              }
+            ],
+            "done": true
+          }
+        },
+        {
+          "close_stream": {
+            "status": {
+              "code": "11",
+              "message": "Partition boundaries are misaligned."
+            },
+            "continuation_tokens": [
+              {
+                "partition": {
+                  "row_range": {
+                    "start_key_closed": "",
+                    "end_key_open": "0000000000000001"
+                  }
+                },
+                "token": "close-stream-token-1"
+              },
+              {
+                "partition": {
+                  "row_range": {
+                    "start_key_closed": "0000000000000001",
+                    "end_key_open": "0000000000000002"
+                  }
+                },
+                "token": "close-stream-token-2"
+              }
+            ]
+          }
+        }
+      ],
+      "results": [
+        {
+          "record": {
+            "change_stream_mutation": {
+              "row_key": "0000000000000000",
+              "type": "USER",
+              "source_cluster_id": "source-cluster-id",
+              "commit_timestamp": "2022-07-01T00:00:00Z",
+              "tiebreaker": 100,
```

Comment:
how is tiebreaker used?

Reply:
It'll be used by the beam connector in case two ChangeStreamMutations have the same commit timestamp